src/Billing/Controllers/StripeController.cs

```diff
@@ -104,6 +104,10 @@ public async Task<IActionResult> PostWebhook([FromQuery] string key)
                 _logger.LogWarning("Getting test events in production.");
                 return new BadRequestResult();
             }
+            else if (_hostingEnvironment.IsEnvironment("QA"))
+            {
+                _logger.LogWarning("Getting events in QA.");
```

Comment:
Seems like if we're not expecting events in QA we should return a `new BadRequestResult()` here as well. OR, if we're getting `live` events in QA, etc. This feels wrong logging a warning w/o taking some appropriate action

Reply:
oh, yep. This was debugging code that I forgot to remove for those emails from Stripe over the weekend. This has been removed.